cfg/acq.cfg

```diff
@@ -311,10 +410,12 @@ lt:
 		//How often we run the scaler
 		servo_interval=1;
 		//PID loop P term
-		P=0.0002;
+		P=0.002;
+		//Phased Trigger PID loop P term
+		phased_P=0.002;
```

Comment:
do we want the other PID terms for the phased trigger? Could define separate servo sections for both to be a bit cleaner...